main/src/ca/uwaterloo/flix/language/phase/PatMatch.scala

```diff
@@ -17,11 +17,10 @@
 package ca.uwaterloo.flix.language.phase
 
 import ca.uwaterloo.flix.api.Flix
-import ca.uwaterloo.flix.language.ast.Symbol.EnumSym
+import ca.uwaterloo.flix.language.ast.*
```

Comment:
Are we mixing stars and underscores now?

Reply:
IntelliJ did this.

Reply:
Yeah. I think we just roll with it for now.